installers/msi-language/ActiveState/Rollbar.cs

```diff
@@ -73,13 +73,35 @@ private static void SetRollbarReportingUser(string id, string userName)
             person.UserName = userName;
             RollbarLocator.RollbarInstance.Config.Person = person;
         }
+    }
+}
+
+public class RollbarReport
+{
+    private static RollbarReport _instance;
```

Comment:
Why record the RollbarReport? Just make this a boolean called `NoFurtherReports` or something.